src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -257,6 +257,18 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
         @Config.RangeDouble(min = 0, max = 1200)
         @Config.LangKey("config." + MOD_ID + ".general.regenCrops")
         public double regenCrops = 12;
+
+        @Config.Comment("Add iron ore dictionary to wrought iron items?")
+        @Config.LangKey("config." + MOD_ID + ".general.oreDictIron")
+        public boolean oreDictIron = false;
+
+        @Config.Comment("Add plate ore dictionary to sheet items?")
+        @Config.LangKey("config." + MOD_ID + ".general.oreDictPlate")
+        public boolean oreDictPlate = false;
+
+        @Config.Comment("Disable the ability to create ingot piles?")
+        @Config.LangKey("config." + MOD_ID + ".general.disableIngotPile")
+        public boolean disableIngotPile = false;
```

Comment:
This is something I am really reluctant to do - it opens the door to a whole range of modularity based problems - i.e. disabling and enabling specific parts of the mod which causes other parts to have to sanity check that they are enabled. I do not think that is a viable approach to take with this mod, mainly due to the size and complexity.

Also, this seems like a bandaid fix to an actual problem - TFC ingot piles don't work that well. So instead of just slapping a disable config on them, can we actually make them work better?

Reply:
I tho this enhancement was because it was asked for a config so other mods can take over if modpacker wants to? see https://github.com/TerraFirmaCraft/TerraFirmaCraft/issues/436

Reply:
Yes it was. I disagree with the sentiment, for the reasons I mentioned above. Just because somebody asks for it doesn't mean it should get implemented.

Reply:
Removed commit and changes